### PR TITLE
build: update deps and ramda functions

### PR DESCRIPTION
**Pull request by crash7**

### CODE REVIEW

File: src/remove.js [hunk 1]
```diff
@@ -7,17 +7,19 @@
...
 const generateKey = require('./generate-key');
 const addReturnValues = require('./return-values');
+const pipeP = pipeWith(andThen);
 
 /**
  * @private
  */
-const removeAndUnwrapAttributes = deleteItem => pipeP(apply(deleteItem), unwrapProp('Attributes'));
+const removeAndUnwrapAttributes = deleteItem =>
+  pipeP([apply(deleteItem), unwrapProp('Attributes')]);
```
Comment (crash7) [inline src/remove.js:22]: @nfantone does this makes sense to you? I can move the pipeP to a util folder before merging.

Comment (nfantone) [reply src/remove.js:22]: Makes sense to me. Although, in hindsight, I would probably use a regular `async` function now.

**Review by nfantone — APPROVED** [review]

File: src/remove.js (hunk 1 above)
Comment (crash7) [reply src/remove.js]: Something like [this](https://github.com/flybondi/flynamo/pull/51/commits/010863e6cb7bc5e9f33fb03011b05592fafa198f)?

**Comment by crash7** [comment]
@nfantone would you check the last change whenever you have time? https://github.com/flybondi/flynamo/pull/51/commits/010863e6cb7bc5e9f33fb03011b05592fafa198f

**Review by nfantone — APPROVED** [review]

**Pull request merged**